.travis.yml

```diff
@@ -2,7 +2,6 @@ language: node_js
 node_js:
   - stable
   - 8
-  - 6
```

Comment:
Do we really drop support for this?

Reply:
We do not. Let's re-add that.

Reply:
@NoNameProvided serious? Even angular drop node 6 for development, do we really need that? Im against.

Reply:
I also read that express dropped support if I recognize correctly

Reply:
I can be bought in on dropping node 6, but then just go with `stable` and `lts/*` instead not? Anyway we can do this after release.